services/worker-manager/src/api.js

```diff
@@ -165,3 +165,32 @@ builder.declare({
   await this.WorkerType.remove({name}, true);
   return res.reply();
 });
+
+builder.declare({
+  method: 'get',
+  route: '/workertypes',
+  query: {
+    continuationToken: /./,
+    limit: /^[0-9]+$/,
+  },
+  name: 'listWorkerTypes',
+  title: 'List All WorkerTypes',
+  stability: APIBuilder.stability.experimental,
+  description: [
+    'TODO',
```

Comment:
Let's add a description and get this landed!

Only other thing is an output schema. It should just be an array of the workertype-full objects. You can use `$ref` to set that up.